Motun-Ankara.md

```diff
@@ -0,0 +1,9 @@
+# Ankara, Turkey
```

Comment:
Yes, it is. I strongly suggest you visit Istanbul, one of the most beautiful cities in the world!